lib/pages/chat/input_bar/focus_suggestion_list.dart

```diff
@@ -22,7 +23,7 @@ class FocusSuggestionList extends StatefulWidget {
 
 class _FocusSuggestionListState extends State<FocusSuggestionList> {
   static const _suggestionMaxVisible = 4.5;
-  static const _scrollAnimationDuration = Duration(milliseconds: 300);
+  static const _scrollAnimationDuration = Duration(milliseconds: 10);
```

Comment:
Why 10ms and not another number?

Reply:
0.3s is too big, for example when user long press the up/down arrow, Another reason is that the lib don't support `jumpTo`. So it's should be as small as possible to look better

Reply:
But why `10ms` and not `20ms` or `15ms` or even some other number? :))